plugins/org.locationtech.udig.ui/src/org/locationtech/udig/ui/operations/PropertyParser.java

```diff
@@ -9,43 +9,44 @@
  */
 package org.locationtech.udig.ui.operations;
 
-import org.locationtech.udig.internal.ui.UiPlugin;
 
 import org.eclipse.core.runtime.IConfigurationElement;
 import org.eclipse.core.runtime.Platform;
+import org.locationtech.udig.internal.ui.UiPlugin;
+import org.locationtech.udig.internal.ui.operations.OpFilterPropertyValueCondition;
```

Comment:
I assume the commit doesn't contain any change you made locally. Please correct me if I'm wrong,  it looks like you moved OpFilterPropertyValueCondition in an internal package bit this pull request doesn't has a move of this class

Reply:
Missed thad, will push in a bit :)

Reply:
@sschulz92 thanks for adressing this. I we would merge this changese right now , we would have two classes of OpFilterPropertyValueCondition in different packages. Hav you used "git mv" or just added the new file? To fix it it might help to git rm the old one and git will detect it as a move finally, I guess